docs/admin/accessing-the-api.md

```diff
@@ -42,7 +42,7 @@ The input to the authentication step is the entire HTTP request, however, it typ
 just examines the headers and/or client certificate.
 
 Authentication modules include Client Certificates, Password, and Plain Tokens,
-and JWT Tokens (used for service accounts).
+Bootstrap Tokens, and JWT Tokens (used for service accounts).
```

Comment:
Bootstrap tokens don't use JWT at all -- they are just random values.  The JWT tokens actually are encoded and signed tokens used for authentication.  We do use JWS for doing the configmap signing, but that is unrelated and not part of the authentication flow.